packages/rspack-test-tools/src/processor/normal.ts

```diff
@@ -23,7 +23,10 @@ export class RspackNormalProcessor extends BasicTaskProcessor<ECompilerType.Rspa
 	constructor(protected _normalOptions: IRspackNormalProcessorOptions) {
 		super({
 			compilerType: ECompilerType.Rspack,
-			findBundle: (context, options) => options.output?.filename,
+			findBundle: (context, options) => {
+				const filename = options.output?.filename;
+				return typeof filename === "string" ? filename : undefined;
```

Comment:
Is this used?

Reply:
cc @LingyuCoder

Reply:
Yes, this looks good to me, findBundle() is used to get the asset file to run, if filename is not a string, an error will be caused

Reply:
Yes. The type of `options.output?.filename` is now changed `string | function`, but `findBundle` is defined to return `string[] | string | void` in [here](https://github.com/web-infra-dev/rspack/blob/c6b58310cfe9db5fc6ef2013177f1f8303d1ce47/packages/rspack-test-tools/src/processor/basic.ts#L26).